### DISCUSSION TITLE
"Time" lifebar parameter for Time Over, Draw, and Double KO.

### DISCUSSION THREAD

**Discussion by UrsoManso** [ikemen-engine/Ikemen-GO#1870] [Feature Request] [answered]
### Is your feature request related to a problem? Please describe.

Those parameters use the "KO.time" parameter value to establish how many ticks after the rounds is over it takes for them to show up. Which can lead to some weird interactions.
### Describe the solution you'd like

Would be nice if they had their own "time" parameter so you could customize how many ticks it would take for each of these animations and sounds to show up. If omitted it would use the "KO.time" value like the current default behavior.

**Comment by potsmugen** [comment, marked as answer]
Implemented.
https://github.com/ikemen-engine/Ikemen-GO/pull/3449
